ui/apps/platform/src/services/NetworkService.js

```diff
@@ -467,6 +467,7 @@ export function getExternalNetworkFlows(
     entityId,
     namespaces,
     deployments,
+    sinceTimestamp,
```

Comment:
An observation, not a change request, about functions with many arguments (especially JS instead of TS).

Here is change from ordered arguments to named properties for `getListQueryParams({ searchFilter, sortOption, page, perPage });` function that you see below about line 480
https://github.com/stackrox/stackrox/pull/12746/files#diff-1bdd625dddb142ce420ab6fe119c5762c807a393a2e53fb2db68b7932000bcb9L200-L218

Reply:
Are you saying you think a single object parameter would be an improvement here? I definitely agree that a lot of these service calls are a mess. `NetworkService` in general could you a refactor/ts upgrade imo.